manifests/julia/traefik-services.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: traefik-external-np
```

Comment:
This NodePort Service has been moved to `manifests/traefik` so it is not needed here any more. There was nothing site-specific about it, so I moved it.

Reply:
ok, removed entire file as discussed.